Source/Code/EventHandler/Relativity PageInteractionEventHandler/PageInteractionEventHandler.cs

```diff
@@ -41,8 +38,8 @@ public override Response PopulateScriptBlocks()
 
 			//Load Javascript and CSS from an existing Relativity Custom Page
 
-			//Let's get the url to our custom pages so we can pull in script/css pages from there
-			String applicationPath = getApplicationPath(this.Application.ApplicationUrl);
+			//Retrieve the URL to the custom page where your external files are stored.  Use your custom page application's GUID.
+			string applicationPath = this.Helper.GetUrlHelper().GetRelativePathToCustomPages(new Guid("439e0ca3-cfbf-4940-a868-c9cd70d0368d"));
```

Comment:
if this is the application guid, I would create a variable with name applicationGuid and then use that variable. That way the user using the template will be clear on what guid to use. 
Right now I am confused what guid this refers to.

Reply:
I see your comment about application guid on the line above. but a named variable would be more clear I think.

Reply:
Updated